From: Simon McVittie <smcv@debian.org>
Date: Wed, 19 Aug 2015 21:27:54 +0100
Subject: Don't check sphinx-build version

It isn't really used for anything anyway, and sphinx-build's
output changed in 1.3 (<https://bugs.debian.org/792715>) resulting
in FTBFS.

Forwarded: not-needed, upstream have removed the Sphinx docs
---
 cmake/OpenCVDetectPython.cmake | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/cmake/OpenCVDetectPython.cmake b/cmake/OpenCVDetectPython.cmake
index d02b759..acd69e8 100644
--- a/cmake/OpenCVDetectPython.cmake
+++ b/cmake/OpenCVDetectPython.cmake
@@ -120,15 +120,9 @@ if(PYTHON_EXECUTABLE)
   if(BUILD_DOCS)
     find_host_program(SPHINX_BUILD sphinx-build)
     if(SPHINX_BUILD)
-        execute_process(COMMAND "${SPHINX_BUILD}"
-                        OUTPUT_QUIET
-                        ERROR_VARIABLE SPHINX_OUTPUT
-                        OUTPUT_STRIP_TRAILING_WHITESPACE)
-        if(SPHINX_OUTPUT MATCHES "Sphinx v([0-9][^ \n]*)")
-          set(SPHINX_VERSION "${CMAKE_MATCH_1}")
+          set(SPHINX_VERSION "(unknown version)")
           set(HAVE_SPHINX 1)
           message(STATUS "Found Sphinx ${SPHINX_VERSION}: ${SPHINX_BUILD}")
-        endif()
     endif()
   endif(BUILD_DOCS)
 endif(PYTHON_EXECUTABLE)
